config/jobs/extension-shoot-oidc-service/extension-shoot-oidc-service-e2e-kind.yaml

```diff
@@ -0,0 +1,48 @@
+presubmits:
```

Comment:
Side-note: later on, you might want to add a periodic job on master as well just like in g/g. :)

Reply:
Yes, the step I added is a dummy step that is not yet implemented (I have a poc ready).
https://github.com/gardener/gardener-extension-shoot-oidc-service/blob/master/hack/test-e2e-provider-local.sh
I will try it in a PR and if everything is OK I will also submit a periodic job.